Sources/ProjectSpec/Target.swift

```diff
@@ -219,7 +218,7 @@ extension Target: NamedJSONDictionaryConvertible {
 
     public init(name: String, jsonDictionary: JSONDictionary) throws {
         self.name = jsonDictionary.json(atKeyPath: "name") ?? name
-        self.productName = jsonDictionary.json(atKeyPath: "productName")
+        productName = jsonDictionary.json(atKeyPath: "productName")
```

Comment:
just removing `self.`? should we do this on the line above as well (`self.name`) ?

Reply:
SwiftFormat did this. 
self.name is still required as name is a parameter